src/components/Avatar/Avatar.visual.tsx

```diff
@@ -24,7 +24,7 @@ visualize('Avatar', () => {
           src={IMG_SRC}
           onLoad={async () => {
             await delay(500);
-            done();
+            return done();
```

Comment:
shouldn't it be `await done()`? I don't think that `onLoad` expects a promise to be returned

Reply:
it's actually not anyone of them.
I'm trying to figure out why async tests are flaky with this version of storybook